eng/pipelines/common/templates/build-job.yml

```diff
@@ -45,7 +45,6 @@ jobs:
 
       steps:
       - script: $(_buildScript)
-                -ci
```

Comment:
why?

Reply:
The `-ci` flag adds a suffix to the version in the Maestro build promotion pipeline. Without it, the suffix is `-dev`, which might be more appropriate for standalone experiments.

Reply:
-dev is for local scenarios, we shouldn't be using that in CI

Reply:
Ok, reverting this change.